.changeset/tender-balloons-knock.md

```diff
@@ -0,0 +1,9 @@
+---
+'@module-federation/webpack-bundler-runtime': patch
+'@module-federation/enhanced': patch
+'@module-federation/runtime': patch
+'@module-federation/runtime-tools': patch
+'@module-federation/sdk': patch
+---
+
+feat: enhanced
```

Comment:
changeset needs a more detailed description

Reply:
yeah , i have removed it